_data/tools.json

```diff
@@ -1,4 +1,13 @@
 [
+      {
+      "title": "ZeroThreat",
+      "url": "https://zerothreat.ai",
+      "owner": "ZeroThreat",
+      "license": "Free",
+      "platforms": "SaaS",
+      "note": "ZeroThreat is the fastest web app and API security scanner providing DAST capabilities providing modern solutions for modern web applications, and it is free to use.",
```

Comment:
"fastest web app and API security scanner" - I think thats too salesy...

Reply:
also chaining "providing" twice reads kinda awkwardly